apollo-runtime/src/main/java/com/apollographql/apollo/ApolloClient.java

```diff
@@ -195,14 +192,10 @@ public Builder httpCache(@Nonnull ResponseCacheStore cacheStore, @Nonnull Evicti
     /**
      * Set the configuration to be used for normalized cache.
      *
-     * @param cacheStore       the cache store to use
-     * @param cacheKeyResolver the cache key resolver
      * @return The {@link Builder} object to be used for chaining method calls
      */
-    public Builder normalizedCache(@Nonnull CacheStore cacheStore,
-        @Nonnull CacheKeyResolver<Map<String, Object>> cacheKeyResolver) {
-      this.cache = new RealCache(checkNotNull(cacheStore, "cacheStore == null"),
-          checkNotNull(cacheKeyResolver, "cacheKeyResolver == null"));
+    public Builder normalizedCache(@Nonnull Cache cache) {
```

Comment:
Sorry confused, why do we need to expose `RealCache`. Can't user just call 'apolloClient.cache().clearAll()' or even `apolloClient.clearCache()`?